### ISSUE TITLE
Blows up on Server Context

### ISSUE THREAD

**Issue by mattsah** [slashtrace/slashtrace#5]
Interesting project, looking to incorporate it into my framework, but right now the only error I'm really seeing is from slashtrace itself.  It looks like it's trying to `getServer()` from $context, and inject it into a table.  When running from CLI, the `$_SERVER` var contains `$_SERVER['argv']` which is an array, thus, one of the values is an array which is throwing an error for the plates template because it's trying to escape an array which passes directly to `htmlentities()`.  In your table template, perhaps it is worth checking to see if the value is an array at least and recursing?  Include tables within tables.  Or, perhaps at that point, if it is **not a string** you should just `print_r()` and return the variable as a string to be escaped.

```
<div class="tab-pane" data-tab-group="context" data-tab="server">
    <?php $this->insert("partials/context/table", [
        "data" => $context->getServer()
    ]); ?>
</div>
```

You can replicate this, I believe just by setting $_SERVER['whatever'] = array(); then triggerin an error.

**Comment by victorstanciu** [comment]
@mattsah Fixed in f9283d1c5cbdc163840d46ae9f3760b527ae51c4. You can see it working in the Server tab on the [demo page](https://slashtrace.com/demo.php) (final row).

Thanks a lot!

**Comment by mattsah** [comment]
Looks hot.  Good on you!
